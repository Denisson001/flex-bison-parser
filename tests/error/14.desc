Compilation error - invalid slice assignment rule