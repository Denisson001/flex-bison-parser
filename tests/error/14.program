{
    a_s = "0123456789";
    a_s[0:5] = a_s[0:5];
}