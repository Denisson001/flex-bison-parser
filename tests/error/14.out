Parser: —Åompilation error - syntax error, unexpected ':', expecting ']' near line 3
